huaweicloud/services/hss/resource_huaweicloud_hss_quota.go

```diff
@@ -0,0 +1,436 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product HSS
+// ---------------------------------------------------------------
+// Due to bugs in HuaweiCloud SKD, automatic generation writing is used.
+
+package hss
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/eps/v1/enterpriseprojects"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API HSS POST /v5/{project_id}/quotas/orders
+// @API HSS GET /v5/{project_id}/billing/quotas-detail
+// @API HSS POST /v5/{project_id}/{resource_type}/{resource_id}/tags/create
+// @API HSS DELETE /v5/{project_id}/{resource_type}/{resource_id}/tags/{key}
+// @API BSS GET /v2/orders/customer-orders/details/{order_id}
+// @API BSS POST /v2/orders/suscriptions/resources/query
+// @API BSS POST /v2/orders/subscriptions/resources/unsubscribe
+// @API BSS POST /v2/orders/subscriptions/resources/autorenew/{resource_id}
+// @API BSS DELETE /v2/orders/subscriptions/resources/autorenew/{resource_id}
+func ResourceQuota() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceQuotaCreate,
+		ReadContext:   resourceQuotaRead,
+		UpdateContext: resourceQuotaUpdate,
+		DeleteContext: resourceQuotaDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"version": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"period_unit": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"period": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+			"auto_renew": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": common.TagsSchema(),
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
This field should add `Computed` attribute, cause it will be backfilled in Read function.

Reply:
done.